### PR TITLE
📃 Add section

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Component/Metadata/Section.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Metadata;
+
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+final class Section
```
Comment (lchrusciel) [inline src/Component/Metadata/Section.php:17]: Let's rename it to Routing
```suggestion
final class Routing
```

Calling it a `Section` implies it is a superior concept over the routing, while it is an optional configuration. What is more, I had issues understanding what it configures just by seeing naming. `Routing` seems to be much more readable and straightforward name to me. 

However, we should preserve `section` config, but I would recommend putting it as one of keys under `Routing` attribute

File: src/Component/Metadata/Section.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+{
+    private ?Operations $operations;
+
+    public function __construct(
+        private ?string $name = null,
```
Comment (lchrusciel) [inline src/Component/Metadata/Section.php:22]:
```suggestion
        private ?string $section = null,
```

File: src/Component/Metadata/Section.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        private ?string $routePrefix = null,
```
Comment (lchrusciel) [inline src/Component/Metadata/Section.php:23]: If we rename the class to `Routing`, `route` prefix here seems to be tautology 
```suggestion
        private ?string $prefix = null,
```

File: src/Component/Metadata/Section.php (hunk 1 above)
Comment (loic425) [inline src/Component/Metadata/Section.php:17]:
```suggestion
final class OperationGroup
```

Comment (loic425) [inline src/Component/Metadata/Section.php:17]:
```suggestion
final class OperationGroup
```

**Comment by loic425** [comment]
@lchrusciel I think we can close this one... We handle this in Resource attribute directly. WDYT?

**Pull request closed without merging**